php/8.2.dockerfile

```diff
@@ -27,6 +27,10 @@ RUN apt-get update && \
         curl \
         gnupg2
 
+# Install dev certificates
+COPY certs/* /usr/share/ca-certificates/netlogix/
+RUN echo "netlogix/docker-dev-ca.crt" >> /etc/ca-certificates.conf && update-ca-certificates
```

Comment:
Wouldn't it make more sense to copy only the one certificate instead of the contents of the folder, since these must be configured again explicitly?

Reply:
hehe lars ask the same. When can easily add a short script that adds all certificates from the usr folder to the `ca-certificates.conf` when we add a second cert to the repo